Zend/tests/enum/test.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+GH-7821: Can't use arbitrary constant expressiosn in enum cases
```

Comment:
```suggestion
GH-7821: Can't use arbitrary constant expressions in enum cases
```

Reply:
Ops, that whole file was committed by accident.